File: tests/fixtures/corpus/pii-in-logs/negative.py
# Corpus fixture for pii-in-logs-python: safe logging that must NOT be flagged.
# Guards against the field-name regex matching substrings such as
# 'token_endpoint' or 'email_provider'.


def boot(cfg, request_count):
    logger.info("requests handled: " + str(request_count))
    logger.info(f"auth service online at {cfg.token_endpoint}")
    log.debug(cfg.email_provider)

File: tests/fixtures/corpus/pii-in-logs/negative.ts
// Corpus fixture for pii-in-logs-javascript: safe logging that must NOT be
// flagged. Guards against the field-name regex matching substrings such as
// 'tokenEndpoint' or 'emailProvider', or matching string literals.

interface Settings {
  tokenEndpoint: string;
  emailProvider: string;
}

export function boot(settings: Settings, requestCount: number): void {
  console.log("requests handled: " + requestCount);
  logger.info(`auth service online at ${settings.tokenEndpoint}`);
  log.debug(settings.emailProvider);
  logger.warn("account.password value is redacted before logging");
}
